Doc/library/sqlite3.rst

```diff
@@ -1041,19 +1041,34 @@ Connection objects
          (2, 'broccoli pie', 'broccoli cheese onions flour')
          (3, 'pumpkin pie', 'pumpkin sugar flour butter')
 
-   .. method:: load_extension(path, /)
+   .. method:: load_extension(path, /, *, entrypoint=None)
 
-      Load an SQLite extension from a shared library located at *path*.
+      Load an SQLite extension from a shared library.
       Enable extension loading with :meth:`enable_load_extension` before
       calling this method.
 
+      :param str path:
+
+         The path to the SQLite extension.
+
+      :param entrypoint:
+
+         Optional entry point name.
+         If ``None`` (default),
+         SQLite will try to come up with an entry point name of its own.
+
+      :type entrypoint: str | None
+
       .. audit-event:: sqlite3.load_extension connection,path sqlite3.Connection.load_extension
 
       .. versionadded:: 3.2
 
       .. versionchanged:: 3.10
          Added the ``sqlite3.load_extension`` auditing event.
 
+      .. versionchanged:: 3.12
```

Comment:
Since the parameter is entirely new:

```suggestion
      .. versionadded:: 3.12
```

Reply:
That means the above `versionchanged` is also incorrect; the `sqlite3.load_extension` auditing event was _added_ in 3.10.